LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2021 Ratify Authors.
```

Comment:
This is copied from https://github.com/ratify-project/ratify/blob/dev/LICENSE#L189